playbooks/settings.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Satellite Settings
+  hosts: localhost
+  collections:
+    - redhat.satellite
+  environment:
+    SATELLITE_USERNAME: "{{ satellite_username }}"
+    SATELLITE_PASSWORD: "{{ satellite_password }}"
+    SATELLITE_SERVER_URL: "{{ server_url }}"
+    SATELLITE_VALIDATE_CERTS: false
```

Comment:
If a `main.yml` (or similar) would be used, those settings could be skipped when using `include_tasks`